Copy to SD Card root directory to update - Unified Menu Material theme/config.ini

```
@@ -193,14 +187,14 @@ pause_feedrate: X6000 Y6000 Z6000 E600
 
 ### Manual Level Points Edge distance (mm)
 # distance in mm, Feedrates in mm/min
-# Leveling Edge distance format  [pause_pos: X<distance in mm> Y<distance in mm>]
+# Leveling Edge distance format  [level_edge_distance: <distance from edge in mm>]
 # Leveling Z Position format     [level_z_pos: Z<position in mm>]
 # Leveling Z raise format        [level_z_raise: Z<distance in mm>]
 # Leveling feed Rate format      [ X<feedrate mm/min> Y<feedrate mm/min> Z<feedrate mm/min>]
-level_edge_distance: X20 Y20
-level_z_pos:0.2
-level_z_raise:10
-level_feedrate: X6000 Y6000 Z6000
+level_edge_distance:20
+level_z_pos:0.2f
+level_z_raise:10f
```

The config routine was made similar to g-codes parser. It extracts these values from text/char array with `strtod` function. it does not require `f` marker in the end. 
This is similar to parsing g-codes which also does not have the `f` marker
like `G1 X10.5 Y11.23 Z0.45`.

```suggestion
level_z_pos:0.2
level_z_raise:10
```
Many users who are not familiar with any type of programming/coding get confused with the `f` in the end.